### ISSUE TITLE
run-jetty documentation

### ISSUE THREAD

**Issue by p-b-west** [sunng87/ring-jetty9-adapter#8]
The document string for run-jetty in ring-jetty-adapter starts as follows:
  "Start a Jetty webserver to serve the given handler according to the
  supplied options:

  :configurator - a function called with the Jetty Server instance
  :port         - the port to listen on (defaults to 80)
"

The document string for run-jetty in ring-jetty9-adapter starts:
  "Start a Jetty webserver to serve the given handler according to the
supplied options:

:port - the port to listen on (defaults to 80)
"

In other words, the :configurator description is missing.  However, options are still tested for :configurator and processed accordingly.

**Comment by mvarela** [comment]
A question about this, how is the `:configurator` option meant to be used? The server is created before being passed to it, and I haven't found a way to alter the configuration after the server creation. In particular, I need to set the URI compliance to `LEGACY` mode.

**Comment by jimpil** [comment]
The server has been created, but not started yet. Here is an example: 

```clj
(let [ring-handler ...
      conn-stats (ConnectionStatistics.)
      opts {:configurator (fn [^Server s] (.addBeanToAllConnectors s conn-stats))}]
(run-jetty handler opts))
```

**Comment by mvarela** [comment]
I was trying to do something along those lines, but I haven't found a way to actually get the `HttpConfiguration` off of a `Server` or `ServerConnector` object in order to modify it. Would it make sense to pass it as an option to `run-jetty` so it's applied on the server creation?

**Comment by jimpil** [comment]
A little bit of searching led me [here](https://stackoverflow.com/questions/28339724/customizing-httpconfiguration-of-jetty-with-spring-boot). Long story short, for your use-case it goes like this:

- Sever.getConnectors()
- ServerConnector.getConnectionFactory()
- HttpConnectionFactory.getHttpConfiguration()
- HttpConfiguration.setUriCompliance()

Hope this helps...

**Comment by mvarela** [comment]
Man, you're a life-saver! Thank you so much!

**Comment by jimpil** [comment]
@sunng87  you might want to close this

**Comment by sunng87** [comment]
Thank you @jimpil
